core/src/main/scala/io/odin/formatter/Formatter.scala

```diff
@@ -10,6 +12,7 @@ trait Formatter {
 
 object Formatter {
 
-  val simple: Formatter = (msg: LoggerMessage) => s"${msg.toString}. Message: ${msg.message()}"
+  val simple: Formatter = (msg: LoggerMessage) => p"${msg.toString}. Message: ${msg.message()}"
+  val default: Formatter = (msg: LoggerMessage) => p"${msg.timestamp.t.F} [${msg.threadName}] ${msg.level.show} ${msg.position.packageName}.${msg.position.fileName}:${msg.position.line} - ${msg.message()}"
```

Comment:
It's missing stack trace information in case if throwable is there

Reply:
Would we want that on the same row or throw a newline in there? Personally I would not print the exception here at all but instead make it the responsibility of whoever is logging to put it in the message (And remove the throwable from the log functions)